lib/middleman-prismic/preview.rb

```diff
@@ -0,0 +1,22 @@
+module Middleman
+  module Prismic
+    class Preview
+      def initialize(app, options={})
+        @app = app
+      end
+
+      def call(env)
+        req = ::Rack::Request.new(env)
+        if req.path =~ /^\/preview/
```

Comment:
How about using `%r(^/preview)` here, to avoid the need to escape the `/`?

Reply:
awesome! 4303531